src/main/groovy/grails/plugin/cache/web/filter/PageFragmentCachingFilter.java

```diff
@@ -557,18 +567,22 @@ protected void logRequestDetails(HttpServletRequest request, ContentCacheParamet
 			return;
 		}
 
-		log.debug("{}...", message);
-		log.debug("    method = {}", request.getMethod());
-		log.debug("    requestURI = {}", request.getRequestURI());
-		log.debug("    forwardURI = {}", WebUtils.getForwardURI(request));
-		if (WebUtils.isIncludeRequest(request)) {
-			log.debug("    includeURI = {}", request.getAttribute(WebUtils.INCLUDE_REQUEST_URI_ATTRIBUTE));
-		}
-		log.debug("    controller = {}", cacheParameters.getControllerName());
-		log.debug("    action = {}", cacheParameters.getActionName());
+		try {
```

Comment:
Added try catch as this throws exception in test due to `InvokerHelper.invokeMethod(request, "getFormat", null))`